README.md

```diff
@@ -1 +1,254 @@
+- Choose images, text, icons, fonts.
+- Set up the repository and tools has been done in feature-one, and feature-two
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client Side / Front-End</summary>
+  <ul>
+    <li><a href="https://www.w3.org/html/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server Side / Back-End</summary>
+  <ul>
+    <li><a href="https://www.json.org/json-en.html">JSON</a></li>
+    <!-- <li><a href=""> - </a></li>
+    <li><a href=""> - </a></li> -->
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Tools <a name="tools"></a>
+
+  <details>
+    <summary>Code Convention, Code Analysis</summary>
+      <ul>
+        <li><a href="https://eslint.org/">ESLint</a></li>
+        <li><a href="https://webhint.io/">Webhint</a></li>
+        <li><a href="https://stylelint.io/">Stylelint</a></li>
+        <li><a href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?hl=en">Lighthouse</a></li>
+        <li><a href="https://www.npmjs.com/package/npm-check">node_modules checker</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>Version Control, CI/CD, Hosting Service</summary>
+      <ul>
+        <li><a href="https://pages.github.com/">Github Pages</a></li>
+        <li><a href="https://github.com/features/actions">Github Actions</a></li>
+        <li><a href="https://git-scm.com/">Git</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>IDE, Desktop Apps, Other Tools</summary>
+      <ul>
+        <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+        <li><a href="https://desktop.github.com/">Github Desktop</a></li>
+        <li><a href="https://www.behance.net/">Behance</a></li>
+      </ul>
+  </details>
+
+### Key Features <a name="key-features"></a>
+
+- Add new Book
+- Remove Book
+- Single page application
+- Dynamic page
+- Redux 
+- Styling
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Coming Soon]
+
+## 🚀 Live Recording <a name="live-demo"></a>
+
+[Coming Soon]
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git version 2.38.x
+- node.js version > 12.x
+- IDE
+- browser (chrome, firefox, edge, safari)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Javascript-capston
+
+  git https://github.com/faisalakb/Space-Travelers
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Javascript-capston
+  node npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+run live server
+
```

Comment:
- [OPTIONAL] To start the development server, you can use the `npm start` command. This is more reliable and would most likely display the correct content. 👍 😎